Sources/VGSShowSDK/UIElements/VGSImageView/VGSImageViewModel.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  VGSImageViewModel.swift
+//  VGSShowSDK
+
+import Foundation
+
+/// Protocol describing VGS Show Image ViewModel.
+internal protocol VGSImageViewModelProtocol: VGSViewModelProtocol {
+    /// Value.
+    var value: VGSShowImageContent? { get set }
+
+    /// Change value completion block.
+    var onValueChanged: ((VGSShowImageContent?) -> Void)? { get set }
+
+    /// Error completion block.
+    var onError: ((VGSShowError) -> Void)? { get set }
+}
+
+/// `VGSImageViewModel` holds image view model logic.
+internal class VGSImageViewModel: VGSImageViewModelProtocol {
+    
+    /// View type.
+    var viewType: VGSShowViewType = .image
+    
+    /// Reference to the binded view.
+    weak var view: VGSBaseViewProtocol?
+    
+    /// Decoding path.
+    var decodingContentPath: String = ""
+    
+    /// Decoding content mode.
+    var decodingContentMode: VGSShowDecodingContentMode = .image(.rawData(.base64))
```

Comment:
Why we harcode decoding content mode two times, her and in `VGSImageView` class

Reply:
I will remove it from `VGSImageView` since it is not needed there.